include/nuttx/mm/iob.h

```diff
@@ -108,7 +120,7 @@ struct iob_s
 #endif
   unsigned int io_pktlen; /* Total length of the packet */
 
-  uint8_t  io_data[CONFIG_IOB_BUFSIZE];
+  FAR uint8_t *io_data;
```

Comment:
let's keep it as before if CONFIG_IOB_ALIGNMENT equal 1

Reply:
We have a better solution, discard this PR. Please refer to https://github.com/apache/incubator-nuttx/pull/6834